Tutorial Series/Introduction to Financial Python/Tutorial11 Linear Algebra.html

```diff
@@ -51,34 +63,31 @@
 </pre>
 
 <h2>Matrix Multiplication</h2>
-\(X_{ij}\) refers to a specific value in row \(i\) and column \(j\) of a matrix \(X\). For example, \(X_{23}\) is the number in the second row and third column of \(X\).
 
-If A is an \(m \times n\) matrix and B is an \(p \times q\) matrix, and \(n=p\), we can multiply A with B, which is written as \(AB\). Please note that \(AB \neq BA\) in general i.e. matrix multiplication is <strong>not commutative</strong>. If \(q \neq m\), we cannot multiply B with A to get \(BA\) to begin with.
+How are two matrices multiplied? Suppose \(X = AB\). Each entry \(x_{ij}\) of matrix \(X\) is the <a href="#scalar">scalar product</a> of row \(i\) from matrix \(A\) with column \(j\) from matrix \(B\). This is best illustrated with an example:
 
-Consider this matrix product:
 \[ AB = \begin{pmatrix}
-a_{11}&amp;a_{12} \\
-a_{21}&amp;a_{22}\\a_{31}&amp;a_{32}
+a_{11} & a_{12} \\
+a_{21} & a_{22} \\
+a_{31} & a_{32}
 \end{pmatrix}
 \begin{pmatrix}
-b_{11}&amp;b_{12} \\
-b_{21}&amp;b_{22}
+b_{11} & b_{12} \\
+b_{21} & b_{22}
 \end{pmatrix} = \begin{pmatrix}
-x_{11}&amp;x_{12} \\
-x_{21}&amp;x_{22} \\
-x_{31}&amp;x_{32}
+x_{11} & x_{12} \\
+x_{21} & x_{22} \\
+x_{31} & x_{32}

```

Comment:
you may need "&amp; " as this is a HTML page, & doesn't render.

Reply:
The & disappears in the website.

Are the vectors meant to be stack on top or next to each other?

\[ v_1 = \begin{pmatrix} 1 \\ 2 \\ 3 \end{pmatrix} \qquad
   v_2 = \begin{pmatrix} 2 \\ 2 \\ 2 \end{pmatrix} \qquad
   v_3 = \begin{pmatrix} 3 \\ 1 \\ 1 \end{pmatrix} \]

can be combined to produce a matrix:
\[ m = \begin{pmatrix}
1 &amp; 2 &amp; 3 \\
2 &amp; 2 &amp; 2 \\
3 &amp; 1 &amp; 1

Reply:
Actually both & and & amp; (without space) rendered correctly.
The 3 vectors are meant to be next to each other.

Unfortunately the website doesn't support \begin{align*} from the amsmath package.
Is it possible for someone to install it? @ShallweXiaowei 
Otherwise the equations can't be aligned at the = sign.